run_configs/jules/build_jules.py

```diff
@@ -13,26 +13,37 @@
 from fab.steps.compile_fortran import compile_fortran
 from fab.steps.find_source_files import find_source_files, Exclude
 from fab.steps.grab.fcm import fcm_export
-from fab.steps.grab.prebuild import grab_pre_build
 from fab.steps.link import link_exe
 from fab.steps.preprocess import preprocess_fortran
 from fab.steps.root_inc_files import root_inc_files
+from fab.tools import Ifort, Linker, ToolBox
 
 logger = logging.getLogger('fab')
 
 
+class MpiIfort(Ifort):
```

Comment:
Dang, good point. I believe the original code had `mpifort`, and I jumped to the wrong conclusion (`mpiifort` :) ), so I replaced it with Ifort :(

For now, I've added a todo #312 to build_jules.py, though it will affect all examples there, and do a proper implementation there, I hope that's ok.